Assets/UI/PartialScreens/citystates.lua

```diff
@@ -430,6 +430,14 @@ function Close()
         UI.PlaySound("CityStates_Panel_Close");
     end
 
+    local localPlayerID = Game.GetLocalPlayer();
+    if (localPlayerID ~= -1) then
+    	local localPlayer = Players[localPlayerID];
+    	if (localPlayer ~= nil and localPlayer:GetInfluence() ~= nil and not localPlayer:GetInfluence():IsGivingTokensConsidered()) then
+    		localPlayer:GetInfluence():SetGivingTokensConsidered(true);
+    	end
+    end
```

Comment:
The indentation of this block is not correct, we need 2 spaces (here it's a mix between spaces and tabs)